src/os/bluestore/BlueStore.cc

```diff
@@ -20458,6 +20441,16 @@ void BlueStore::set_allocation_in_simple_bmap(SimpleBitmap* sbmap, uint64_t offs
   sbmap->set(offset >> min_alloc_size_order, length >> min_alloc_size_order);
 }
 
+BlueStore::BlobRef BlueStore::ExtentDecoderPartial::decode_create_blob(
```

Comment:
So far both decode_create_blob() implemnentations are the same - do you plan to have them so different in the future that this would need dedicated implementation rather than just control the behavior through params?

Reply:
OK, got the rationales from "recovery speed-up" PR. Indeed having virtual decode_create_blob() makes sense then.

But my above question about rationales behind using templates stil stands..